### PR TITLE
python3.pkgs.phik: fix build

### LINKED ISSUE #172420: python39Packages.phik build failure
### Steps To Reproduce
```
 nix-shell -p '(python39Packages.phik.overrideAttrs (oldAttrs: { doCheck = false; doInstallCheck = false; }))'
```

### Build log
```
building '/nix/store/sadfdrx0ls5b515wg3n702bb242h007g-python3.9-phik-0.12.2.drv'...
Sourcing python-remove-tests-dir-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing python-remove-bin-bytecode-hook.sh
Sourcing setuptools-build-hook
Using setuptoolsBuildPhase
Using setuptoolsShellHook
Sourcing pip-install-hook
Using pipInstallPhase
Sourcing python-imports-check-hook.sh
Using pythonImportsCheckPhase
Sourcing python-namespaces-hook
unpacking sources
unpacking source archive /nix/store/8vvdkz9lpk3zfxvxf1czr4xl3zg5a7dm-phik-0.12.2.tar.gz
source root is phik-0.12.2
setting SOURCE_DATE_EPOCH to timestamp 1647881119 of file phik-0.12.2/PKG-INFO
patching sources
configuring
no configure script, doing nothing
building
Executing setuptoolsBuildPhase
Traceback (most recent call last):
  File "/opt/revr/tmp/nix-build-python3.9-phik-0.12.2.drv-0/phik-0.12.2/nix_run_setup", line 8, in <module>
    exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\\r\\n', '\\n'), __file__, 'exec'))
  File "setup.py", line 20, in <module>
    from skbuild import setup as sk_setup
ModuleNotFoundError: No module named 'skbuild'
error: builder for '/nix/store/sadfdrx0ls5b515wg3n702bb242h007g-python3.9-phik-0.12.2.drv' failed with exit code 1;
       last 10 log lines:
       > configuring
       > no configure script, doing nothing
       > building
       > Executing setuptoolsBuildPhase
       > Traceback (most recent call last):
       >   File "/opt/revr/tmp/nix-build-python3.9-phik-0.12.2.drv-0/phik-0.12.2/nix_run_setup", line 8, in <module>
       >     exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\\r\\n', '\\n'), __file__, 'exec'))
       >   File "setup.py", line 20, in <module>
       >     from skbuild import setup as sk_setup
       > ModuleNotFoundError: No module named 'skbuild'
       For full logs, run 'nix log /nix/store/sadfdrx0ls5b515wg3n702bb242h007g-python3.9-phik-0.12.2.drv'.

```

### Notify maintainers
@mweinelt @trofi @jonringer @melsigl 

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
 - system: `"x86_64-linux"`
 - host os: `Linux 3.10.0-1160.62.1.el7.x86_64, Red Hat Enterprise Linux Server, 7.9 (Maipo), nobuild`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.8.0`
 - channels(alex): `"nixpkgs"`
 - nixpkgs: `/u/alex/.nix-defexpr/channels/nixpkgs`
```

### PR DESCRIPTION
**Pull request by KAction**
Closes: #172420

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by KAction** [comment]
@alexvorobiev You are welcome to check and provide feedback. I made it build, and run basic sanity checks, but I have no clue what it is supposed to actually do.

**Comment by KAction** [comment]
@alexvorobiev Thanks for suggestion. @SuperSandro2000 Can you review, please?

File: pkgs/development/python-modules/phik/default.nix [hunk 1]
```diff
@@ -50,8 +52,11 @@ buildPythonPackage rec {
   nativeBuildInputs = [
     cmake
     ninja
+    scikit-build
   ];
 
+  pythonImportCheck = [ "phik" ];
+
   postInstall = ''
   rm -r $out/bin
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/phik/default.nix:61]:
```suggestion
    rm -r $out/bin
```

Comment (KAction) [reply pkgs/development/python-modules/phik/default.nix]: Can do, but this is how it was before me. I try to make PR as small as possible.

**Pull request merged**